### PR TITLE
cHTTP has been renamed to isahc

### PR DESCRIPTION
**Pull request by vorot93**

### CODE REVIEW

**Comment by berkus** [comment]
This seems to work and pulls entire body.

The hyper-client seems to mess up the body by doing some decode/encode behind the back? Haven't figured out yet what exactly happens.

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ edition = "2018"
 default = ["native-client", "middleware-logger"]
 native-client = ["curl-client", "wasm-client"]
 hyper-client = ["hyper", "runtime", "runtime-raw", "runtime-tokio" ]
-curl-client = ["chttp"]
+curl-client = ["isahc"]
```
Comment (yoshuawuyts) [inline Cargo.toml]: This is a breaking change we can't merge until we're comfortable releasing a 2.0. I feel like we may need to revert this and log it so we don't forget to update it at a later time (:

Comment (yoshuawuyts) [reply Cargo.toml]: Ah nevermind, I totally misread. We're good here!

**Comment by CarlosLanderas** [comment]
How can we fix apps depending on surf version 1.0.1 with chttp if you can't merge this because is a breaking change? Thanks!

**Comment by vorot93** [comment]
This PR has been merged and is there to stay

**Comment by CarlosLanderas** [comment]
Is there any plan to release a new version? I've seen the last one is 1.0.1 and it doest not have the chttp read body fix right? Thanks

**Pull request merged**